src/text/test/like.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { like } from "../like";
+
+const testCases: Array<[string, string, boolean]> = [
+  ["texto", "te", true],
```

Comment:
the "_" character means any char (1 time)
suggested test:
["texto", "te_to", true],
["texto", "_xto", false],

Reply:
Added test cases in commit [dca3e85](https://github.com/genexuslabs/web-standard-functions/pull/16/commits/dca3e85663b64d2baf658799765d9c9080a16446)